OpenOversight/app/templates/base.html

```diff
@@ -41,142 +39,142 @@
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="{{ url_for("auth.change_dept") }}">Change Default Department</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="{{ url_for("auth.logout") }}">Log Out</a>
+                  </li>
+                </ul>
               </li>
-              <li>
-                <a class="navbar-link" href="{{ url_for("main.get_started_labeling") }}">Volunteer</a>
+            {% else %}
+              <li class="nav-item">
+                <a class="nav-link" href="{{ url_for("auth.register") }}">Register</a>
               </li>
-              <li>
-                <a class="navbar-link" href="{{ url_for("main.about_oo") }}">About</a>
+              <li class="nav-item">
+                <a class="nav-link" href="{{ url_for("auth.login") }}">Log In</a>
               </li>
-              {% if current_user and current_user.is_administrator %}
-                <li>
-                  <a class="navbar-link" href="{{ url_for("auth.get_users") }}">Users</a>
-                </li>
-              {% endif %}
-            </ul>
-            <ul class="nav navbar-nav navbar-right">
-              {% if current_user and current_user.is_authenticated %}
-                <li class="dropdown">
-                  <a href="#" class="navbar-link dropdown-toggle" data-toggle="dropdown">Account <b class="caret"></b></a>
-                  <ul class="dropdown-menu">
-                    <li>
-                      <a class="navbar-link"
-                         href="{{ url_for('main.profile', username=current_user.username) }}">Profile</a>
-                    </li>
-                    <li>
-                      <a class="navbar-link" href="{{ url_for("auth.change_password") }}">Change Password</a>
-                    </li>
-                    <li>
-                      <a class="navbar-link" href="{{ url_for("auth.change_email_request") }}">Change Email</a>
-                    </li>
-                    <li>
-                      <a href="{{ url_for("auth.change_dept") }}">Change Default Department</a>
-                    </li>
-                    <li>
-                      <a class="navbar-link" href="{{ url_for("auth.logout") }}">Log Out</a>
-                    </li>
-                  </ul>
-                </li>
-              {% else %}
-                <li>
-                  <a class="navbar-link" href="{{ url_for("auth.register") }}">Register</a>
-                </li>
-                <li>
-                  <a class="navbar-link" href="{{ url_for("auth.login") }}">Log In</a>
-                </li>
-              {% endif %}
-            </ul>
-          </div>
+            {% endif %}
+          </ul>
         </div>
       </div>
     </nav>
     {% with messages = get_flashed_messages() %}
       {% if messages %}
```

Comment:
🔬 Do we need the `if` here?

Reply:
Do you mean the `if messages` block? It's needed because the outer container still adds margins and other styles, even if there are no messages

Reply:
That's what I was talking about, but it seems we're good!